convex/world.ts

```diff
@@ -224,10 +223,6 @@ export const activePlayers = query({
       .withIndex('active', (q) => q.eq('worldId', world._id).eq('active', true))
       .collect();
     for (const player of players) {
-      const location = await ctx.db.get(player.locationId);
-      if (!location) {
-        throw new Error(`Invalid location ID: ${player.locationId}`);
-      }
```

Comment:
I totally missed this when reviewing, but down on L250/245 `location` is still referenced. It must not warn on missing browser globals. TIL